src/content/en/updates/2018/07/chrome-69-media-updates.md

```diff
@@ -0,0 +1,311 @@
+    });
+
+### Available for Origin Trials
+
+In order to get feedback from web developers, the HDCP Policy Check API is
+available as an [Origin Trial] in Chrome 69 for Desktop (Chrome OS, Linux, Mac,
+and Windows). You will need to [request a token], so that the feature would be
+automatically enabled for your origin for a limited period of time, without the
+need to enable the experimental "Web Platform Features" flag at
+`chrome://flags/#enable-experimental-web-platform-features`.
+
+[Intent to Experiment](https://www.google.com/url?q=https://groups.google.com/a/chromium.org/forum/%23!topic/blink-dev/ITzZ_yx4bF8&sa=D&ust=1530528558634000&usg=AFQjCNFc5O4gFlE5rBSytmc73mGgqCyNgA) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5652917147140096) &#124;
+[Chromium Bug](https://crbug.com/709348)
+
+## MSE PTS/DTS compliance {: #pts }
+
+Buffered ranges and duration values are now reported by Presentation Time Stamp
+(PTS) intervals, rather than by Decode Time Stamp (DTS) intervals in [Media
+Source Extensions (MSE)].
+
+At the very beginnings of MSE, Chrome’s implementation was tested against WebM
+and MP3, some media stream formats where there was no such distinction between
+PTS and DTS. And it was working fine… until ISO BMFF (aka MP4) was added. This
+frequently contains out-of-order presentation versus decode time streams (eg,
+for codecs like H.264) causing DTS and PTS to differ. That caused Chrome to
+report (usually just slightly) different buffered ranges and duration values
+than expected. But here it is: this new behavior will roll out gradually in
+Chrome 69 and makes its MSE implementation compliant with the [MSE
+specification].
+
+<figure>
+  <img src="/web/updates/images/2018/07/pts-dts.png"
+       alt="PTS/DTS">
+  <figcaption>
+    <b>Figure 2.</b>
+    PTS/DTS
+  </figcaption>
+</figure>
+
+This change affects `MediaSource.duration` (and consequently
+`HTMLMediaElement.duration`), `SourceBuffer.buffered` (and consequently
+`HTMLMediaElement.buffered)`, and `SourceBuffer.remove(start, end)`.
+
+If you’re not sure which method is used to report buffered ranges and duration
+values, you can go to the internal `chrome://media-internals` page and search for
+"ChunkDemuxer: buffering by PTS" or  "ChunkDemuxer: buffering by DTS" in logs.
+
+[Intent to Implement](https://groups.google.com/a/chromium.org/d/msg/blink-dev/I6oGJLym-Tk/Z46le9l4CQAJ) &#124;
+[Chromium Bug](https://crbug.com/718641)
+
+## Handling of media view intents on Android Go {: #media_intent_handler }
+
+[Android Go] is a lightweight version of Android designed for entry-level
+smartphones. To that end, it does not necessarily ship with some media-viewing
+applications, so if a user tries to open a downloaded video for instance, they
+won’t have any applications to handle that intent.
+
+To fix this, Chrome 69 on Android Go now listens for media-viewing intents so
+users can view downloaded audio, videos and images.
```

Comment:
Comma after videos.

Are you saying Chrome takes the place of the missing viewing applications? That point could be clearer.

Reply:
Done.
Thanks!